teps/draft-20210126-pipeline-task-display-name.md

```diff
@@ -65,6 +65,22 @@ To be determined.
 Pipeline writers can specify a display name for any/all tasks, if supplied that display name
 will be rendered in UI rather then the machine readable `name` field value.
 
+#### Example
+```
+kind: PipelineRun
+spec:
+  pipelineSpec:
+    tasks:
+      - name: echo-message
+         displayName: Echo Message
```

Comment:
(nit): the indentation would not validate

Reply:
Thanks. Fixed.